src/tests/cmocka/test_responder_cache_req.c

```diff
@@ -28,10 +28,12 @@
 #include "db/sysdb.h"
 #include "responder/common/cache_req/cache_req.h"
 
+#define FILES_ID_PROVIDER "files"
+#define LDAP_ID_PROVIDER "ldap"
 #define TESTS_PATH "tp_" BASE_FILE_STEM
 #define TEST_CONF_DB "test_responder_cache_req_conf.ldb"
 #define TEST_DOM_NAME "responder_cache_req_test"
-#define TEST_ID_PROVIDER "ldap"
+#define TEST_ID_PROVIDER LDAP_ID_PROVIDER
```

Comment:
Since you defined both FILES_ID_PROVIDER and LDAP_ID_PROVIDER then I think you can just delete this macro and replace it with LDAP_ID_PROVIDER?

Reply:
Done